### DISCUSSION TITLE
https://ecust-leap.github.io/cise/abroad/2015/yuchen

### DISCUSSION THREAD

**Discussion by giscus** [ECUST-Leap/ecust-leap.github.io#198] [评论区]
# https://ecust-leap.github.io/cise/abroad/2015/yuchen

GPA以及语言成绩是申请研究生（至少是硕士）的基本面，也许是最重要的指标。

https://ecust-leap.github.io/cise/abroad/2015/yuchen

**Comment by ycsh-w** [comment]
Hi大家好，我就是这篇文章的作者，以及这本华理飞跃手册的技术负责人&项目经理。欢迎大家在这个评论区找我聊天！
